pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -257,10 +227,9 @@ func newNetworkAddonsForCR(cr *hcov1alpha1.HyperConverged) *networkaddons.Networ
 			Labels: labels,
 		},
 		Spec: networkaddons.NetworkAddonsConfigSpec{
-			Multus:          &networkaddons.Multus{},
-			LinuxBridge:     &networkaddons.LinuxBridge{},
-			KubeMacPool:     &networkaddons.KubeMacPool{},
-			ImagePullPolicy: v1.PullPolicy(NetworkAddonsImagePullPolicy),
+			Multus:      &networkaddons.Multus{},
```

Comment:
Are these required?  Preferably, we should have nothing.  If they are required, @phoracek can you add `omitempty` and have the network-operator default?

Reply:
They are required. Our operator gives users a way to deploy additional components based on their needs. Addons operator won't deploy anything if the CR is empty. We cannot set them enabled by default because of otherwise there will be no way to disable them. Why is it a problem?

Reply:
Makes sense.  We'll leave it then